stubs/app/Actions/Jetstream/CreateTeam.php

```diff
@@ -27,9 +27,15 @@ public function create($user, array $input)
 
         AddingTeam::dispatch($user);
 
-        return $user->ownedTeams()->create([
+        $team = $user->ownedTeams()->create([
             'name' => $input['name'],
             'personal_team' => false,
         ]);
+
+        $user->forceFill([
```

Comment:
Rethinking this for the failed tests:

```suggestion
        $user->switchTeam($team);
```

Reply:
Trying now!

Reply:
@joelbutcher That seems to solve the failing tests! But lastly i cant wrap my head around why StyleCI is bugging us. That file looks perfectly fine to me. Do you see any thing?

Reply:
@OscarKolsrud it's the white space after 

```php
         $personalTeam->forceFill(['personal_team' => true])->save();

```

Reply:
Fixed! All tests passing! I think this is now ready to merge :)